<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterTime</class>
 <widget class="QDialog" name="RegisterTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(5, 14, 43)</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>531</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : white;
color : black;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : transparent;
color : white;
font: 700 22pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Change Registration Time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font: 700 22pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Open Time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font: 700 22pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Close Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="message">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>523</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;color : green;font-size:20px;</string>
   </property>
   <property name="text">
    <string>Saved successfully</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>822</x>
     <y>510</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
border-radius : 20px;
background : white;
color : black;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="updSem">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>17</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;color : green;font-size:20px;</string>
   </property>
   <property name="text">
    <string>Updated successfully</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;
    border-radius: 10px;
font-size : 20px ;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
color:white;
font: 16pt &quot;Arial&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Start-Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
color:white;
font: 16pt &quot;Arial&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>End-Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="EndDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;
    border-radius: 10px;
font-size : 20px ;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
